cmd/skopeo/sync.go

```diff
@@ -25,6 +25,19 @@ type imagesBySource struct {
 	SourceCtx        *types.SystemContext
 }
 
+type registryCfg struct {
```

Comment:
Maybe we should name it `registrySyncCfg` to make it clear/avoid clashes in the future.